tensorflow/compiler/aot/tfcompile_main.cc

```diff
@@ -136,6 +136,10 @@ int main(int argc, char** argv) {
 
   tensorflow::string usage = tensorflow::tfcompile::kUsageHeader;
   usage += tensorflow::Flags::Usage(argv[0], flag_list);
+  if (argc > 1 && !strcmp(argv[1], "--help")) {
```

Comment:
We conventionally use `absl::string_view::operator==` instead of strcmp.  So something like

```
  if (argc > 1 && absl::string_view(argv[1]) == "--help")
```

Reply:
Thanks for mention this api.